specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/firewallPolicy.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "NetworkManagementClient",
     "description": "The Microsoft Azure Network management API provides a RESTful set of web services that interact with Microsoft Azure Networks service to manage your network resources. The API has entities that capture the relationship between an end user and the Microsoft Azure Networks service.",
-    "version": "2020-06-01"
+    "version": "2020-08-01"
```

Comment:
you cannot change the api-version in this spot - you need to create a new api-version altogether.  Please coordinate with Mikhail on what this version should be and where it is appropriate to work on it.